Create Simplified Chinese Version of README.rst

LShun opened the pull request:
## Create Simplified Chinese Version of README.rst
### Description

Hello, I translated the README.rst to Simplified Chinese using Google Translate and my own understanding in Simplified Chinese language. I hope this can help people who are not fluent in English but are fluent in Chinese to be able to understand the project easier and faster. There are some words that I didn't convert to Chinese either because they are special names, or specific links which does not have a translated version ( for example at line 68: LICENSE_ an NOTICE_ ). The file name is README-zh.rst, but I don't mind if its changed, its just for easier initial identification.
### Status

done, ready for review
### Checklist (tick everything that applies)
- [ ] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [v] Documentation
- [ ] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

LShun committed:
Create Simplified Chinese Version of README.rst

Hello, I translated the README.rst to Simplified Chinese using Google Translate and my own understanding in Simplified Chinese language. I hope this can help people who are not fluent in English but are fluent in Chinese to be able to understand the project easier and faster. There are some words that I didn't convert to Chinese either because they are special names, or specific links which does not have a translated version ( for example at line 68: LICENSE_ an NOTICE_ ). The file name is README-zh.rst, but I don't mind if its changed, its just for easier initial identification.

Thank you.

tonybaloney commented:
I'm going to struggle to code review this one! I can't read chinese. Open to anyone for review

watermelo commented:
I think the translation is not good enough to understand and exist some typos.

LShun commented:
I see, thanks. Would you mind to specify the typo's and the parts which are
hard to understand so I can correct them?

On Tuesday, November 15, 2016, watermelo notifications@github.com wrote:

> I think the translation is not good enough to understand and exist some
> typos.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/apache/libcloud/pull/907#issuecomment-260539876, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/AJNIUQNCiJBtbhPLPXo-inVdJlKVmgfuks5q-StHgaJpZM4KXC5h
> .

watermelo commented on README-zh.rst:
```diff
@@ -0,0 +1,71 @@
+.. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
+        :target: https://coveralls.io/github/apache/libcloud?branch=trunk
+
+Apache Libcloud 是一个Python 的库，隐藏不同云端供着的应用程序接口（API's) 和允许您通过统一和易于使用的API和 Python library管理不同的云端资源。
```
Apache Libcloud是一个Python库，它隐蔽了不同云资源API层的差异，并允许您通过统一且易于使用的API来管理不同的云资源。

LShun replied:
Thank you, I edited it with your correction.

watermelo commented on README-zh.rst:
```diff
@@ -0,0 +1,71 @@
+* **容器** - 虚拟化服务容器 (``libcloud.container.*``)
+
+
+Apache Libcloud 是一个 Apache 的计划, 往 <http://libcloud.apache.org> 看以获得跟多资料
```
更多资料，不是跟多

LShun replied:
Thank you, I edited it with your suggestion.

watermelo commented on README-zh.rst:
```diff
@@ -0,0 +1,71 @@
+执照
+==
+
+Apache Libcloud是根据Apache 2.0被许可证授权的。 更多有关详情，请参阅 LICENSE_ 和 NOTICE_ 文件。
```
Apache Libcloud是被Apache 2.0许可证授权的。

LShun replied:
Thank you, I edited it with your correction.

watermelo commented on README-zh.rst:
```diff
@@ -0,0 +1,71 @@
+
+* **计算** - 云服务器和块存储 - 服务例子： Amazon EC2 and Rackspace
+  Cloud Servers (``libcloud.compute.*``)
+* **存储** - 云对象存储 and 内容交付网络(CDN-Content Delivery Network ）  - 服务例子： Amazon S3 and Rackspace
```
and改成和

LShun replied:
Thank you, I edited it with your correction.

watermelo reviewed (commented):
hi~ @LShun, Correction is as follows, hope be modified.

LShun committed:
Update 2 for README-zh.rst

Edited a few typography errors and mistakes made using @watermelo 's suggestion. Thank you @watermelo.
